docs/users/exchanges.md

```diff
@@ -0,0 +1,240 @@
+# Exchanges
+
+This example shows a possible flow of how an exchange would use Hermez.
+
+## Pre-requisites
+-  Exchange already has some Hermez accounts for each trading token (`HEZEx-ETH`, `HEZEx-DAI`,...). 
+-  User has an Hermez account (`HEZUser-ETH`)
+![](exchanges/Exchange-init.jpg)
+
+Existing accounts are regular Hermez accounts, consisting of an L2 account linked to an Ethereum account where funds can be withdrawn.
+
+## User Transfers Tokens to Exchange
+A User wants to transfer 10 ETH from his Hermez account to the exchange for the first time. To do so, the user requests to transfer some funds via some sort of front end provided by the exchange.
+After the request is done, the exchange provides the address of an internal Hermez account where the user can deposit his tokens. This account doesn't have an Ethereum counterpart account, 
+and thus the creation is really cheap. Once the user has received the L2 account address, he can perform the transfer normally.
+In the meantime, the exchange is monitoring the status of this account, and once the user transfer is completed, the exchange can transfer the funds to its main account. This process is depicted in the diagram below.
+
+The creation of this user account by the exchange is only done once. 
+
+![](exchanges/Exchange-transfer.jpg)
+
+### Flow 
+1. User requests to do a transfer to the exchange from his Hermez account using some front-end.
```

Comment:
Please discard if this isn't the correct terminology for this instance:
"1. User requests to do a transfer to the exchange from his Hermez account using a front-end framework."